components/Workout/index.js

```diff
@@ -0,0 +1,101 @@
+import styled from "styled-components";
+import ModalDelete from "../ModalDelete";
+import Link from "next/link";
+import WorkoutPreview from "../WorkoutPreview";
+import { useState } from "react";
+import { toast } from "react-toastify";
+
+export default function Workout({ workouts, onDeleteWorkout, exercises }) {
+  const [showModal, setShowModal] = useState(false);
+  const [workoutIdToDelete, setWorkoutIdToDelete] = useState(0);
```

Comment:
```suggestion
  const [workoutIdToDelete, setWorkoutIdToDelete] = useState(null);
```
Using 0 might be confusing because 0 could be a valid id potentially